model/search/ResultsWatcher.php

```diff
@@ -103,7 +103,7 @@ protected function addIndex(DeliveryExecutionInterface $deliveryExecution)
                 self::INDEX_TEST_TAKER => $user->getIdentifier(),
                 self::INDEX_TEST_TAKER_FIRST_NAME => UserHelper::getUserFirstName($user, true),
                 self::INDEX_TEST_TAKER_LAST_NAME => UserHelper::getUserLastName($user, true),
-                self::INDEX_TEST_TAKER_NAME => UserHelper::getUserName($user, true),
+                self::INDEX_TEST_TAKER_NAME => UserHelper::getUserLabel($user),
```

Comment:
Should we consider rename the constant too?

Reply:
IMO this class is very much unstable. I would not like to make any additional changes (this include const name) before we make order in here.

Reply:
Ok :)

Reply:
Maybe I will create new index for this called 'label' and I will keep name intact (it will get firstname and lastname together)

Reply:
Sounds good :)

Reply:
done